### ISSUE TITLE
Avoid Thumbnail preview in folder

### ISSUE THREAD

**Issue by htwsaaraub** [owncloud/files_mediaviewer#178]
owncloud: 10.3.2 (stable)
Media Viewer: 1.0.1

Hello,

I would like to request a configuration switch for disable the small preview thumbnails in folder View. 
This Process consums all processing power just by entering the folder. 
It would really suffice if the preview just generate when the user is klicking the File for gallery preview.
This would avoid to generate previews for the whole folder when the user just want to see specific files.

**Comment by Matthiasfranck** [comment]
A better solution would be if the previews could be pregenerated.

**Comment by htwsaaraub** [comment]
As we primary use external shares in Owncloud this would also generate a preview thumbnail for every different LDAP user which Logs into Owncloud. 
So, for enterprise use you should be able to disable Feature to save Performance. 
At this time the Previews already generated when a Users scrolls through the Folder. This generates  lots of Processes which eats cpu. 
Better would be to generate the Preview at the Moment when a user opens the Picture in the Viewer.

**Comment by sebascreen** [comment]
Yes, it should be possible to disables thumbnails of image files but after click onthe image file it should be viewed. Right now behaviour after set 'enable_previews' => false, ownCloud wants to download file.
